src/lib/containers/QueryWrapperMenu.tsx

```diff
@@ -15,11 +15,15 @@ import {
   isGroupByInSql,
```

Comment:
I noticed that this component doesn't update when navigating with the back and forward buttons on the browser, I think this might have to get wrapped in withRouter to trigger componentDidUpdate on URL changes?

Reply:
@leem42 - I have created[https://sagebionetworks.jira.com/browse/PORTALS-1188]( https://sagebionetworks.jira.com/browse/PORTALS-1188) to track this